src/_includes/install/windows.md

```diff
@@ -44,7 +46,27 @@ To upgrade the Dart SDK:
 C:\> choco upgrade dart-sdk
 ```
 
+### Uninstall using Chocolatey {:.no_toc}
+
+To uninstall the Dart SDK, use [Chocolatey][Chocolatey].
+Chocolatey requires [elevated permissions].
+
+1. Launch [Powershell][] with elevated permissions.
+
+1. Use the following command.
+
+   ```ps
+   C:\> choco uninstall dart-sdk
+   ```
+
+1. Remove the Dart configuration files from your home directory.
+
+   ```dart
+   C:\> Remove-Item -Recurse -Force ^
+        -Path $env:LOCALAPPDATA\.dartServer,$env:APPDATA\.dart,$env:APPDATA\.dart-tool
+   ```
+
 [elevated permissions]: https://www.thewindowsclub.com/elevated-privileges-windows
-[command prompt]: https://www.thewindowsclub.com/how-to-run-command-prompt-as-an-administrator
+[Powershell]: https://www.thewindowsclub.com/how-to-open-an-elevated-powershell-prompt-in-windows-10
```

Comment:
Is there an official Microsoft source we can use for this instead?

Reply:
Unfortunately, no.